testing/test_collection.py

```diff
@@ -642,3 +642,114 @@ def test___repr__():
         """)
         reprec = testdir.inline_run("-k repr")
         reprec.assertoutcome(passed=1, failed=0)
+
+
+COLLECTION_ERROR_PY_FILES = dict(
```

Comment:
Just a tip: sometimes these type of metadata is better put into a session fixture. :wink: